docs/RegistryCoordinator.md

```diff
@@ -0,0 +1,325 @@
+* Updates each quorum's `quorumUpdateBlockNumber` to the current block
+* For any quorums where the Operator no longer meets the minimum stake, they are deregistered (see `deregisterOperator` above).
+
+*Requirements*:
+* Pause status MUST NOT be set: `PAUSED_UPDATE_OPERATOR`
+* See `updateOperators` above
+* `quorumNumbers` MUST be an ordered array of quorum numbers, with no entry exceeding the current `quorumCount`
+* All `quorumNumbers` MUST correspond to valid, initialized quorums
+* `operatorsPerQuorum` and `quorumNumbers` MUST have the same lengths
+* Each entry in `operatorsPerQuorum` MUST contain an order list of the currently-registered Operator addresses in the corresponding quorum
+* See [`StakeRegistry.updateOperatorStake`](./registries/StakeRegistry.md#updateoperatorstake)
+
+#### `updateSocket`
+
+```solidity
+function updateSocket(string memory socket) external
+```
+
+Allows a registered Operator to emit an event, updating their socket.
+
+*Effects*:
+* Emits an `OperatorSocketUpdate` event
+
+*Requirements*:
+* Caller MUST be a registered Operator
+
+---
+
+### System Configuration
+
+These methods are used by the Owner to configure the `RegistryCoordinator`:
+* [`createQuorum`](#createquorum)
+* [`setOperatorSetParams`](#setoperatorsetparams)
+* [`setChurnApprover`](#setchurnapprover)
+* [`setEjector`](#setejector)
+
+#### `createQuorum`
+
+```solidity
+function createQuorum(
+    OperatorSetParam memory operatorSetParams,
+    uint96 minimumStake,
+    IStakeRegistry.StrategyParams[] memory strategyParams
+) 
+    external
+    virtual 
+    onlyOwner
+```
+
+Allows the Owner to initialize a new quorum with the given configuration. The new quorum is assigned a sequential quorum number.
+
+The new quorum is also initialized in each of the registry contracts.
+
+*Effects*:
+* `quorumCount` is incremented by 1
+* The new quorum's `OperatorSetParams` are initialized (see `setOperatorSetParams` below)
+* See [`BLSApkRegistry.initializeQuorum`](./registries/BLSApkRegistry.md#initializequorum)
+* See [`StakeRegistry.initializeQuorum`](./registries/StakeRegistry.md#initializequorum)
+* See [`IndexRegistry.initializeQuorum`](./registries/IndexRegistry.md#initializequorum)
+
+*Requirements*:
+* Caller MUST be the Owner
+* Quorum count before creation MUST be less than `MAX_QUORUM_COUNT`
+* See [`BLSApkRegistry.initializeQuorum`](./registries/BLSApkRegistry.md#initializequorum)
+* See [`StakeRegistry.initializeQuorum`](./registries/StakeRegistry.md#initializequorum)
+* See [`IndexRegistry.initializeQuorum`](./registries/IndexRegistry.md#initializequorum)
+
+#### `setOperatorSetParams`
+
+```solidity
+function setOperatorSetParams(
+    uint8 quorumNumber, 
+    OperatorSetParam memory operatorSetParams
+) 
+    external 
+    onlyOwner 
+    quorumExists(quorumNumber)
+```
+
+Allows the Owner to update an existing quorum's `OperatorSetParams`, which determine:
+* `maxOperatorCount`: The max number of operators that can be in this quorum
```

Comment:
question unrelated to docs, but wondering if we have testing to cover the correct/desired behavior when reducing this below the current number of operators?

Reply:
@8sunyuan tag for you here